include/openssl/evp.h

```diff
@@ -1078,6 +1093,31 @@ struct evp_pkey_st {
   const EVP_PKEY_ASN1_METHOD *ameth;
 } /* EVP_PKEY */;
 
+struct evp_kem_st {
+  int name_id;
+  const char *type_name;
+  const char *description;
+
+  int (*encapsulate_init)(void *ctx,
+                          void *provkey,
+                          const OSSL_PARAM params[]);
+  int (*encapsulate)(void *ctx,
+                     unsigned char *out,
+                     size_t *outlen,
+                     unsigned char *secret,
+                     size_t *secretlen);
+  int (*decapsulate_init)(void *ctx,
+                          void *provkey,
+                          const OSSL_PARAM params[]);
+  int (*decapsulate)(void *ctx,
+                     unsigned char *out,
+                     size_t *outlen,
+                     const unsigned char *in,
+                     size_t inlen);
+
+} /* EVP_KEM */;
```

Comment:
```suggestion
}; //  EVP_KEM 
```

Reply:
Another case of following convention of the other structures. That said, I don't feel strongly one way or the other. I'll change it if your example is the preferred style.